core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/WorkspaceBillingAccountActor.scala

```diff
@@ -0,0 +1,275 @@
+package org.broadinstitute.dsde.rawls.monitor
+
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import cats.Applicative
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import cats.implicits._
+import com.google.api.services.cloudbilling.model.ProjectBillingInfo
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.dataaccess.slick.{BillingAccountChange, WriteAction}
+import org.broadinstitute.dsde.rawls.dataaccess.{GoogleServicesDAO, SlickDataSource}
+import org.broadinstitute.dsde.rawls.model._
+import org.broadinstitute.dsde.rawls.monitor.migration.MigrationUtils.Implicits._
+import org.broadinstitute.dsde.rawls.monitor.migration.MigrationUtils.Outcome
+import org.broadinstitute.dsde.rawls.monitor.migration.MigrationUtils.Outcome.{Failure, Success}
+
+import java.time.Instant
+import scala.concurrent.duration._
+
+object WorkspaceBillingAccountActor {
+  sealed trait WorkspaceBillingAccountsMessage
+
+  case object UpdateBillingAccounts extends WorkspaceBillingAccountsMessage
+
+  def apply(dataSource: SlickDataSource,
+            gcsDAO: GoogleServicesDAO,
+            initialDelay: FiniteDuration,
+            pollInterval: FiniteDuration): Behavior[WorkspaceBillingAccountsMessage] =
```

Comment:
Typed actors do seem better. My akka-actor knowledge is rusty though. :)

fwiw, Leo moved all actor usage to fs2 streams a while back. See: https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/AsyncTaskProcessor.scala